filesystem/templatecopy.go

```diff
@@ -26,6 +26,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// DelimiterPairKey is key for the delimeter pair in add-on config
+	DelimiterPairKey = "delimiterPair"
+	// ConfigKey is key for the template config in add-on config
+	ConfigKey = "config"
+	// DefaultOpeningDelimiter is default opening delimiter used in golang templates
+	DefaultOpeningDelimiter = "{{"
+	// DefaultClosingDelimiter is default closing delimiter used in golang templates
+	DefaultClosingDelimiter = "}}"
+	// CustomOpeningDelimiter is custom opening delimiter used in golang templates
+	CustomOpeningDelimiter = "[["
```

Comment:
[[ is used in other context, so there is a chance it might conflict in many cases, like say startup command etc.. https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs

Reply:
done